src/adapter/adapter.ts

```diff
@@ -16,14 +15,7 @@ import { resultMapper } from './demo-result.mapper';
 import { priceFilterMapper } from './demo-price-filter-mapper';
 import { EmpathyEndpointsService } from './empathy-endpoints.service';
 
-declare global {
-  interface Window {
-    initX: SnippetConfig;
-  }
-}
-
 export const adapter = new EmpathyAdapterBuilder()
-  .setEnvironment(window.initX.env ?? 'test')
```

Comment:
Great job, this was breaking one of the ways of initialising the project.  However I think this will be an issue later. Right now this keeps working because the `env-selector` component subscribes to the `ExtraParamsChanged` event, and updates the search adapter config. 
However this component won't exist in clients. So the env won't be changed. As we are already working in the adapter, I would probably just try to set this configuration in the `installExtraPlugins` callback of the `InstallXOptions`, with a big TODO reminding us to remove it when the adapter is changed to be stateless.